recipe/build.sh

```diff
@@ -1,11 +1,26 @@
 #!/bin/sh
 set -euo pipefail
 
+# get vtk library for osx-arm64
+VTK_DIR=${PREFIX}
+if [[ "$OSTYPE" == "darwin"* && $OSX_ARCH == "arm64" ]]; then
+    VTK_DIR=${RECIPE_DIR}/vtk
+    conda list -p ${BUILD_PREFIX} > packages.txt
+    cat packages.txt
+    VTK_PACKAGE_VERSION=`grep vtk packages.txt | awk -F ' ' '{print $2}'`
+    export CONDA_SUBDIR=osx-arm64
+		conda create -y -p ${VTK_DIR} --no-deps vtk=${VTK_PACKAGE_VERSION}
+fi
+
 # FindOpenCascade.cmake bundled with OCP references the env var `CONDA_PREFIX`.
 # That is the right prefix when manually running CMake inside a conda env, but
 # the wrong one when using conda-build. Substitute the right prefix inline.
-CONDA_PREFIX="${PREFIX}" cmake -B build -S "${SRC_DIR}/src" \
+CONDA_PREFIX="${PREFIX}" cmake ${CMAKE_ARGS} -B build -S "${SRC_DIR}/src" \
 	-G Ninja \
+	-DCMAKE_PREFIX_PATH="${VTK_DIR}" \
+	-DPython3_FIND_STRATEGY=LOCATION \
```

Comment:
I wonder if these `Python3*` flags are required. Did you try without?

The CMakeLists file being processed here is [this one](https://github.com/CadQuery/pywrap/blob/d3bc7321dab7d82340b4fe16defb2578cf6610de/bindgen/CMakeLists.j2), which uses `find_package(pybind11 REQUIRED)` **without**  calling `find_package(python)` before it.

I don't know which Python hint locations are used by this command.

https://pybind11.readthedocs.io/en/stable/cmake/index.html

Reply:
I tried to remove the FindPython stuff but it fails to build on Linux then, so I added them back ;)